### PR TITLE
Member Profile of Sandra Ashipala added

### LINKED ISSUE #5027: [Community] Member Profile: Sandra Ashipala
**Current Behavior**

@sandramsc has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @sandramsc as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @sandramsc
- Twitter: [sandraamsc](https://twitter.com/sandraamsc)
- LinkedIn: https://www.linkedin.com/in/sandraashipala/
- Meshery Cloud: 
- Link to profile picture:

A detailed explanation of how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by VivekBaldha0712**
**Description**

This PR fixes #5027 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
image

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0c4acb69447097c88eb11e872a4120f0aeae961d at: https://652981af57bf91009b3c603d--layer5.netlify.app

File: src/collections/members/sandra-ashipala/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Sandra Ashipala
+position: Contributor
+image_path: ./sandra-ashipala.webp
+github: sandramsc
+twitter: sandraamsc
+linkedin: sandraashipala
+layer5: aaeb3b36-ac02-450a-bb7e-964ad9d818e4
+location: Berlin, Germany
```
Comment (sandramsc) [inline src/collections/members/sandra-ashipala/index.mdx:9]: Relocated back to Namibia from Germany, please update as current location.
```suggestion
location: Namibia
```

**Review by sandramsc — CHANGES_REQUESTED** [review]
Hi, @abhijeetgauravm thanks for your contribution 🥳, I've left an inline comment to update the current location. Also please take note that the [DCO check ](https://github.com/layer5io/layer5/pull/5039/checks?check_run_id=17686554329)needs to be successful as well.

File: src/collections/members/sandra-ashipala/index.mdx (hunk 1 above)
Comment (VivekBaldha0712) [reply src/collections/members/sandra-ashipala/index.mdx]: hey @sandramsc , I have done the required changes suggested from you. Please have a look

**Comment by l5io** [comment]
🚀 Preview for commit 763fc79cde6f73762c649ecaa3d92344916c3d50 at: https://652a0aa5d80c3460fdc1db37--layer5.netlify.app

**Review by Mohith234 — APPROVED** [review]
Fix the DCO check and we are good to go. LGTM 🚀 @VivekBaldha0712 

// @sandramsc 🙌

**Review by sandramsc — APPROVED** [review]
LGTM!

**Pull request closed without merging**